pfl-basic/src/main/java/org/glassfish/pfl/basic/algorithm/Algorithms.java

```diff
@@ -40,6 +36,7 @@ private Algorithms() {}
      * @return list of objects
      * @deprecated replaced by {@link Arrays#asList(java.lang.Object...)}
      */
+    @Deprecated
```

Comment:
Given
- it was deprecated 5+ years ago - for 4.1 it seems
- we're targeting 5.1 release
- below public `getAnnotationValues` is removed w/o deprecation
maybe we could remove this `list` as well?

Reply:
I thought you will comment this, but I tried to limit changes - this was removed because it was just failing and not used anywhere. But yeah, ok, I agree, I will remove it now, it is better.